packages/connections/src/stores/connections-store.ts

```diff
@@ -175,6 +177,10 @@ export function useConnections(
       connectedConnectionInfo.current &&
       connectedDataService.current
     ) {
+      // Update lastUsed date as now and save the connection.
+      connectedConnectionInfo.current.lastUsed = new Date();
+      void connectionStorage.save(connectedConnectionInfo.current);
```

Comment:
👍 Added a catch and now we show the error on a banner:

https://user-images.githubusercontent.com/1791149/142765693-bd75fb36-5ef1-409a-8b0f-d86a29d2bb34.mp4

This will be more useful when saving a connection is more of the connection flow, since with this saving event we move off of this connect screen so the saving error probably can go unnoticed. We could move displaying it into compass-home, but I think it makes sense to break that into its own work then. I think it makes sense not to block the connecting when a saving error occurs.

Reply:
Makes sense, should we report and log the error?

Reply:
Adding logging for the error 👍